docs/release-notes.md

```diff
@@ -7,15 +7,12 @@ This section outlines key features changes between versions 1.3 and 1.4 for Open
 
 ### Additions
 #### RPS & MPS
```

Comment:
Breaking changes:

-Authentication for Websockets -- that was listed as a known issue last release, that has been resolved this release. JWT required for KVM/SOL connections
- Environment variables removed in MPS
- Vault KeyName changes in RPS (removed profile prefix from key name)
- MQTT!!!!
- Global Credentials removed for MPS
- No default passwords

Reply:
done